apps/yggdrasil_web/web/controllers/character_controller.ex

```diff
@@ -0,0 +1,58 @@
+defmodule YggdrasilWeb.CharacterController do
+  use YggdrasilWeb.Web, :controller
+  use Guardian.Phoenix.Controller
+
+  alias Yggdrasil.Character
+
+  def index(conn, _params, user, _claims) do
+    user_id = user.id
```

Comment:
Yeah, I think this is where documentation is going to be vital.

For example `/characters?filter[user_id]=...' should be used when wanting the specifics user's characters and not rely `/characters` and the users permissions. But this is entirely on the client and the more I think about it I think it's not really an issue. Vilive for example would probably use the filtered form if has a page to manage the user's character and then can show more by removing the filter, it's apparent now that it really is the client's responsibility.

So yeah I agree with you, one API controlled with permissions.

So do we want to do that in this round or accept this in a slightly broken state to collapse this down and get characters/games into master and then tackle permissions. Permissions are going to bring up a lot of questions I think. Purely looking at some guidance here, this PR is getting lengthy.